github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2546,3 +2546,74 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
+    optional: true
+    annotations:
+      testgrid-dashboards: kubevirt-presubmits
+      fork-per-release: "true"
+    cluster: ibm-prow-jobs
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 4h0m0s
+    extra_refs:
+    - base_ref: main
+      org: kubevirt
+      repo: project-infra
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-github-credentials: "true"
+      preset-kubevirtci-quay-credential: "true"
+      preset-pgp-bot-key: "true"
+    max_concurrency: 1
+    name: pull-kubevirt-e2e-k8s-1.26-sev
+    skip_branches:
+      - release-\d+\.\d+
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - |
+          # install yq
+          curl -Lo ./yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+          chmod +x ./yq && mv ./yq /usr/local/bin/yq
+
+          # install sshuttle and kubectl
+          dnf -y install sshuttle kubernetes-client
+          # get kubeconfig and ssh key
+          source ../project-infra/hack/manage-secrets.sh
+          decrypt_secrets
+          extract_secret 'amdKubeConfig' /kubeconfig
+          extract_secret 'amdSEVClusterKey' /ssh-key && chmod 400 /ssh-key
+
+          # login quay.io
+          cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
+
+          # Set up tunnel
+          sshuttle -D -r kubevirt@165.204.53.121 10.216.91.126/32 --ssh-cmd 'ssh -o StrictHostKeyChecking=no -i /ssh-key -p 30026'
+
+          # run test
+          make cluster-sync && FUNC_TEST_LABEL_FILTER='--label-filter=(SEV)' make functest && make cluster-clean || make cluster-clean
```

Comment:
I don't see the PR that introduces the label into the KubeVirt code base, can you give me a pointer where [decorators.go](https://github.com/kubevirt/kubevirt/blob/main/tests/decorators/decorators.go) is modified?